### PR TITLE
Adding a CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by venantius**
Setting expectations for people inside and outside of Airbnb about what we expect in our issues and pull requests.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+*within Airbnb*. It isn't a community style guide, and as such we may reject
+issues or PRs that simply don't fit our internal engineering needs.
+
+## Pull Requests
+
+PRs to the style guide should include a description of the proposal, including
+examples of what "good" and "bad" code would look like under the new rule, as
+well as a rationale for the rule in question. The description of the rule and
+the examples belong in `README.md`; the rationale goes in the corresponding
+section in `rationales.md`
```
Comment (tommydangerous) [inline CONTRIBUTING.md]: Perhaps add instructions to create a link between the 2 files for each rule or guide created.

Comment (venantius) [inline CONTRIBUTING.md]: Great suggestion.

**Comment by venantius** [comment]
Updated, PTAL.

**Comment by ljharb** [comment]
LGTM, some others should check too

**Comment by BMorearty** [comment]
LGTM

**Pull request merged**